src/sentry/api/endpoints/project_rules.py

```diff
@@ -205,7 +205,7 @@ def _compare_lists_of_dicts(
             raw_left = {k: v for k, v in left.items() if k not in keys_to_ignore}
             raw_right = {k: v for k, v in right.items() if k not in keys_to_ignore}
 
-            # TODO (Yash): This code commented below is the corrected logic which accounts for bad key values.
+            # TODO: (Yash): This code commented below is the corrected logic which accounts for bad key values.
```

Comment:
Here the colon char already exists, so there is no need to add it twice :)

Reply:
That one in fact slipped me. Thanks